final/style.css

```diff
@@ -0,0 +1,174 @@
+/**********************************************
+*** GENERAL
+**********************************************/
+
```

Comment:
Need more comments in the .css, particularly for what the colours are given by the # values.